libpod/runtime.go

```diff
@@ -217,18 +218,6 @@ var (
 	}
 )
 
-func getDefaultTmpDir() (string, error) {
```

Comment:
Is this safe to remove in favor of the `GetRootlessRuntimeDir()` function? @giuseppe?

Reply:
When run as root it still uses /run/run/libpod, I put that into the default runtime specification.
When rootless we overide and call the rootless pars ofGetRootlessRuntimeDir.  Bottom line it should work the same way it currently does, except that it uses DRIVER-libpod/tmp for the directories to store the tmp content.